python/ccf/ledger.py

```diff
@@ -169,13 +180,42 @@ def __next__(self):
         except:
             raise StopIteration()
 
-
 class Ledger:
 
-    _filename = None
+    _filenames = []
+    _directory = None
+    _fileindex = 0
 
-    def __init__(self, filename):
-        self._filename = filename
+    def __init__(self, name:str, is_dir:bool == False):
+        if is_dir:
+            contents = os.listdir(name)
+            # Sorts the list based off the first number after ledger_ so that the ledger is verified in sequence 
+            sort = sorted(contents, key=lambda x: int(
+                x.replace(".committed", "").replace("ledger_", "").split("-")[0]))
+
+            for chunk in sort:
+                # Add only the .committed ledgers to be verified 
```

Comment:
It would be great to print out the names of the uncommitted files that are not included here, possibly with `LOG.warning()`.